### DISCUSSION TITLE
Subsonic Playlists

### DISCUSSION THREAD

**Discussion by DANW999** [strawberrymusicplayer/strawberry#897] [Ideas]
Hi,

I know this has already been mentioned in https://github.com/strawberrymusicplayer/strawberry/issues/327 but could I humbly request that this may be reconsidered? Strawberry is such a great music player and my go to player on  my Linux. One of the main things I use it for is playing music from my Subsonic server and the only thing that's missing is the fact I cannot see/play the music playlists that I created on my Subsonic server. Ultrasonic, the android application, supports this feature and it is great to use so I am just wondering if we could possibly get the same feature on Strawberry, Great product and I appreciate all your work!

**Comment by xorander00** [comment]
I second this. It would allow me to dump my other players in favor of Strawberry if it supported Subsonic playlists. Even if just read-only for the first implementation.

**Comment by llucps** [comment]
Yes, that's my case too. I'd love to have subsonic playlists.

**Comment by TheBlackKoala** [comment]
I'm looking into moving to something like subsonic for portability and i can't find a music player that supports subsonic playlists while also being half as usable as strawberry (which is my regular go-to anyhow) so subsonic playlists would really be a gamechanger for me!

**Comment by jonaski** [comment]
https://forum.strawberrymusicplayer.org/category/6/feature-suggestions
